schemes/tropopause_find/tropopause_find.meta

```diff
@@ -115,13 +115,13 @@
   standard_name = tropopause_air_pressure_from_climatology_dataset
   units = Pa
   type = real | kind = kind_phys
-  dimensions = (horizontal_dimension, number_of_months_in_year)
+  dimensions = (horizontal_dimension, number_of_time_slices_in_climatology_dataset)
```

Comment:
I realize we should probably be as explicit as possible and label this a `tropopause_climatology_dataset`:

```suggestion
  dimensions = (horizontal_dimension, number_of_time_slices_in_tropopause_climatology_dataset)
```

The same request holds below and for the equivalent uses  in CAM-SIMA as well.  Thanks!

Reply:
Thanks @nusbaume! I updated throughout to clarify the standard names for the tropopause climatology dataset related variables.

* Updated `tropopause_air_pressure_from_climatology_dataset` -> `tropopause_air_pressure_from_tropopause_climatology_dataset`
* Updated `number_of_time_slices_in_climatology_dataset` -> `number_of_time_slices_in_tropopause_climatology_dataset`
* Updated `tropopause_calendar_days_from_climatology` -> `tropopause_calendar_days_from_tropopause_climatology`